### DISCUSSION TITLE
Developer Profile

### DISCUSSION THREAD

**Discussion by shivarm** [quarkiverse/quarkus-quinoa#469] [General]
@melloware @ia3andy should we create a developer profile? to give credit who active at this project.

**Comment by melloware** [comment]
I haven't seen the other extensions do it but personally I have no problem with is as the more info in the pom.xml the better.

**Reply by shivarm** [reply]
done-https://github.com/quarkiverse/quarkus-quinoa/pull/471

**Comment by shivarm** [comment]
@melloware what company are you working for?
